src/ripple/app/main/Application.cpp

```diff
@@ -1085,16 +1085,16 @@ bool ApplicationImp::setup()
     }
 
     {
-        PublicKey validationPub;
-        SecretKey validationPriv;
+        PublicKey valPublic;
+        SecretKey valSecret;
         std::string manifest;
         if (config().exists (SECTION_VALIDATOR_TOKEN))
         {
             if (auto const token = ValidatorToken::make_ValidatorToken (
                 config().section (SECTION_VALIDATOR_TOKEN).lines ()))
             {
-                validationPriv = token->validationPrivate;
-                validationPub = derivePublicKey (KeyType::secp256k1, validationPriv);
+                valSecret = token->validationPrivate;
```

Comment:
Legacy names I'm not so confident about, particularly if they may have leaked into our API.  Mostly I'm reacting to how our newer code seems to prefer "secret" over "private".  Where it's possible I'd like to continue that trend toward "secret" so we don't end up with an ugly mix.  Just my take on it.